visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +137,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
```

Comment:
@wkentaro: Could you propose a PR (w.r.t. #713) for turning the execute service into an action?
Dave[ agreed on that approach](https://github.com/ros-planning/moveit_ros/pull/713#issuecomment-232848243), but I don't have time for it right now.
I think, having that done too, we can merge this branch.